idl/forward_request.idl.hh

```diff
@@ -0,0 +1,23 @@
+namespace query {
+struct forward_request {
+    enum class reduction_type : uint8_t {
+        count,
+    };
+
+    std::vector<query::forward_request::reduction_type> reduction_types;
+    std::vector<column_id> selected_columns;
```

Comment:
This is redundant with partition_slice::regular_columns.

Or maybe not, if we select a static or primary key column.

Reply:
I will remove this field as it's not used ATM. If anything comes up in the future, we can rethink having it.